nautobot/extras/tests/test_models.py

```diff
@@ -1072,6 +1073,36 @@ def test_remote_url_hostname(self):
         self.repo.remote_url = "http://some-private-host/example.git"
         self.repo.validated_save()
 
+    def test_clone_and_discard_clone_functions(self):
+        """Confirm that the clone() and discard_clone() methods work as expected."""
+        self.repo.remote_url = "https://github.com/nautobot/nautobot.git"
```

Comment:
That's gonna be slow :-) Would be better to use the `create_and_populate_git_repository` helper in `nautobot/extras/tests/git_helper.py`